warehouse/models/gtfs_rt/_gtfs_rt.yml

```diff
@@ -5,28 +5,86 @@ sources:
     database: cal-itp-data-infra
     schema: gtfs_rt
     tables:
-      - name: vehicle_positions
+      - name: calitp_files
```

Comment:
long-term question: do we want to try to document/link back to the jobs that populate these source tables? I have mixed feelings; could be useful for debugging but could also be misleading if they end up out-of-date.

Reply:
I gave it a go, LMK if you want me to remove